### ISSUE TITLE
The "contrast" skin $primary-color does not have sufficient contrast according to WCAG.

### ISSUE THREAD

**Issue by lukasvermeer** [mmistakes/minimal-mistakes#5122]
I was a bit surprised to see my website fail lighthouse accessibility checks due to "lack of contrast" after switching to the "contrast" theme. :-)

The current $primary-color red (#ff0000) fails the WCAG requirements because it does not have sufficient contrast when used on buttons. Using blue (#0000ff) does pass. Please consider using a $primary-color that would let the skin pass the WCAG guidelines. 

If you want me to propose an MR I would be happy to, but I didn't want to impose my colour choices upon others before opening an issue first to discuss. :-)

**Comment by iBug** [comment]
I have no idea about this. I tend to err on the side of caution lest more people complain about the sudden color change.

**Comment by lukasvermeer** [comment]
I guess it depends what people care about more: a sudden color change or WCAG compliance and website accessibility. For me personally, the latter would be far more important. :-)

Perhaps there is a less dramatic color change possible that would also be WCAG compliant.

**Comment by iBug** [comment]
Any chance you can pass the check while keeping its red tone? Like tuning the brightness slightly, such as `#CC0000` or `#FF3333`? If that's possible I'd merge a PR for that.

**Comment by lukasvermeer** [comment]
I think `#B60000` might work. Challenge is this colour is used as primary button background but also the hamburger menu.

**Comment by lukasvermeer** [comment]
<img width="612" alt="Image" src="https://github.com/user-attachments/assets/c437ff7d-b6bf-40da-9996-345c3b50c2fe" />

**Comment by lukasvermeer** [comment]
Shall I open a PR to change [this line](https://github.com/mmistakes/minimal-mistakes/blob/cb140fc5e648c6d808d444a34b6126c43f87d8bd/_sass/minimal-mistakes/skins/_contrast.scss#L8) from:
```$primary-color: #ff0000 !default;```

to:
```$primary-color: #b60000 !default;```

**Comment by iBug** [comment]
Yes, that's fine. Go ahead.
